core/test/financial-templates/ExpiringMultiPartyCreator.js

```diff
@@ -32,9 +32,6 @@ contract("ExpiringMultiPartyCreator", function(accounts) {
     collateralToken = await Token.new("UMA", "UMA", 18, { from: contractCreator });
     registry = await Registry.deployed();
     expiringMultiPartyCreator = await ExpiringMultiPartyCreator.deployed();
-    await registry.addMember(RegistryRolesEnum.CONTRACT_CREATOR, expiringMultiPartyCreator.address, {
```

Comment:
Why remove this from this test?

Reply:
Because I added it to the migrations, so we don't need it there anymore (I don't think).